### PR TITLE
docs/packaging: update the mandatory `homepage` key

### PR DESCRIPTION
**Pull request by silkeh**
## Description

This has become mandatory and is checked by linters, see https://github.com/getsolus/packages/pull/377.

### Submitter Checklist

- [x] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: docs/packaging/packaging-changes.md [hunk 1]
```diff
@@ -11,16 +11,14 @@ This page is meant to serve as a changelog of sorts for the Solus packaging envi
...
-_These items are not mandatory yet, and are subject to change_
+### October
 
 #### Requiring `homepage` in `package.yml`
 
-- The `homepage` key in `package.yml` will become mandatory. If a package does not have a homepage, use the address for the upstream source.
-- We are exploring how to enforce this; for now we will ask nicely.
-
-## 2023
+- The `homepage` key in `package.yml` has become mandatory. If a package does not have a homepage, use the address for the upstream source.
```
Comment (EbonJaeger) [inline docs/packaging/packaging-changes.md:20]: > use the address for the upstream source.

This is ambiguous; does it mean the URL to the tarball? The GitHub repository? Something else?

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Just one thing that I think should be clarified more.

File: docs/packaging/packaging-changes.md (hunk 1 above)
Comment (silkeh) [reply docs/packaging/packaging-changes.md]: Changed to: 

> Use the address to the upstream source repository (eg: the GitHub page) if a package does not have a homepage.

Is that clear?

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! :)

**Pull request merged**